### PR TITLE
[MAINT] Update ruff to 0.8.0

### PR DESCRIPTION
**Pull request by DimitriPapadopoulos**

### CODE REVIEW

File: nilearn/_utils/__init__.py [hunk 1]
```diff
@@ -200,8 +200,8 @@ def all_classes(
 
 
 __all__ = [
-    "_repr_niimgs",
     "CacheMixin",
+    "_repr_niimgs",
```
Comment (Remi-Gau) [inline nilearn/_utils/__init__.py:204]: unrelated: probably worth renaming this function to non private if we expose it as part of the public part of the `_utils` subpackage

**Review by Remi-Gau — APPROVED** [review]

File: nilearn/_utils/__init__.py (hunk 1 above)
Comment (Remi-Gau) [reply nilearn/_utils/__init__.py]: ha thanks!
Sorry I did not necessarily mean to change it in this PR, but I'll take it in any case.

**Review by man-shu — APPROVED** [review]
LGTM!

**Comment by Remi-Gau** [comment]
thanks @DimitriPapadopoulos

**Pull request merged**